cycode/cli/printers/sca_table_printer.py

```diff
@@ -0,0 +1,166 @@
+from collections import defaultdict
+from typing import List, Dict
+
+import click
+from texttable import Texttable
+
+from cycode.cli.consts import LICENSE_COMPLIANCE_POLICY_ID, PACKAGE_VULNERABILITY_POLICY_ID
+from cycode.cli.models import DocumentDetections, Detection, CliError, CliResult
+from cycode.cli.printers.base_printer import BasePrinter
+
+SEVERITY_COLUMN = 'Severity'
+LICENSE_COLUMN = 'License'
+UPGRADE_COLUMN = 'Upgrade'
+REPOSITORY_COLUMN = 'Repository'
+CVE_COLUMN = 'CVE'
+
+PREVIEW_DETECTIONS_COMMON_HEADERS = [
+    'File Path',
+    'Ecosystem',
+    'Dependency Name',
+    'Direct Dependency',
+    'Development Dependency'
+]
+
+
+class SCATablePrinter(BasePrinter):
```

Comment:
can we inherit from TablePrinter? it makes sense?

Reply:
IMHO the methods are so unique per table. but yeah, we could create BaseTablePrinter and move the constructor, print_result, and print_error methods (at least)